pipeline/wj_luigi.py

```diff
@@ -128,12 +129,12 @@ def run(self):
         ''')
         for i in range(5):
             message += '|'.join(map(str, df.values.tolist()[:5][i])) + '\n'
-        '''
 
         msg = EmailMessage()
+        msg["Date"] = datetime.now() -+ timedelta(hours=8)
```

Comment:
Can you explain to me why see jie and you both using the the same -+?

Reply:
Ah, the code is wrong, it suppose to be (datetime.now() - timedelta(hours=8)).
We got discuss together yesterday and there is a mistake that I delete the wrong row while testing.

Reply:
I see. OK.